tests/rkt_exec_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"os"
 	"testing"
 
-	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/steveeJ/gexpect"
```

Comment:
Saw that but since we're probably gonna switch back to the main repo I didn't change it.

I guess if we remember to change it back when we do that I can also put steveeJ's fork in README.